tests/test_user_login.py

```diff
@@ -0,0 +1,65 @@
+"""
+This file holds user login test cases.
+"""
+import unittest
+import json
+from api import create_app
```

Comment:
Where is this method `create_app` that you are importing here in your files? I can't seem to track it @Ogollah

Reply:
api/__init__.py